providers/base/bin/network.py

```diff
@@ -166,27 +167,71 @@ def summarize_cpu(self):
                 avg_cpu = sum_cpu / n
         return avg_cpu
 
+    def find_numa(self, device):
+        """Return the NUMA node of the specified network device."""
+        filename = "/sys/class/net/" + device + "/device/numa_node"
+        try:
+            file = open(filename, "r")
+            node_num = int(file.read())
+        except FileNotFoundError:
```

Comment:
I just realized that FileNotFoundError is referred to a couple times, but is not imported. The script also traps CalledProcessError directly, but that IS explicitly imported.